### DISCUSSION TITLE
Transformar em api, mas o que?

### DISCUSSION THREAD

**Discussion by WoLfulus** [BrasilAPI/BrasilAPI#242] [Q&A]
> Vamos transformar o Brasil em uma API?

Pela chamada do projeto, fiquei imaginando que tipos de APIs seriam essas?

Como sugestão, seria interessante colocar quais APIs estariam no escopo do projeto ou não (mesmo que não implementadas) visto que hoje só existe a de CEP, ou até uma issue pontuando quais fariam sentido existir aqui.

Abraço!

**Comment by NetoBraghetto** [comment]
Seria interessante uma api pra verificar feriados
